src/main/java/ai/deepcode/javaclient/requests/GetAnalysisRequest.java

```diff
@@ -102,47 +102,63 @@ public int hashCode() {
     }
   }
 
-  private static class AnalysisContext {
-    private final String flow;
-    private final String initiator = "IDE";
-    private final String orgDisplayName;
+  private static class AnalysisContextOrg {
+    private final String name;
 
-    public AnalysisContext(String flow, String orgDisplayName) {
-      this.flow = flow;
-      this.orgDisplayName = orgDisplayName;
+    public AnalysisContextOrg(String name) {
+      this.name = name;
     }
 
-    public String getFlow() {
-      return flow;
+    @Override
+    public boolean equals(Object o) {
+      if (this == o) return true;
+      if (o == null || getClass() != o.getClass()) return false;
+      AnalysisContextOrg that = (AnalysisContextOrg) o;
+      return Objects.equals(name, that.name);
     }
 
-    public String getOrgDisplayName() {
-      return orgDisplayName;
+    @Override
+    public int hashCode() {
+      return Objects.hash(name);
     }
 
-    public String getInitiator() {
-      return initiator;
+    @Override
+    public String toString() {
+      return "AnalysisContextOrg{" +
+        "name='" + name + '\'' +
+        '}';
+    }
+  }
+
+  private static class AnalysisContext {
+    private final String flow;
+    private final String initiator = "IDE";
+    private AnalysisContextOrg org;
+
+    public AnalysisContext(String flow, String orgName) {
+      this.flow = flow;
+      this.org = new AnalysisContextOrg(orgName);
```

Comment:
What happens if the org is empty?

Reply:
it's fine. in this case default org will be taken into account